tests/common.sh

```diff
@@ -37,6 +37,29 @@ function wait_clean {
   done
 }
 
+# The argument is 'timeout in seconds'
+function get_uuid () {
+  sleep 5
+  counter=0
+  sleep_time=5
+  counter_max=$(( $1 / sleep_time ))
+  uuid="False"
+  until [ $uuid != "False" ] ; do
+    if [ -z $uuid ]; then
+      uuid=$(kubectl -n my-ripsaw get benchmarks -o jsonpath='{.items[0].status.uuid}')
+      sleep $sleep_time
+      uuid="False"
```

Comment:
This code will never succeed. You obtain the uuid then set it to false two lines later.

Reply:
@aakarshg